src/hotspot/cpu/riscv/globals_riscv.hpp

```diff
@@ -114,6 +114,7 @@ define_pd_global(intx, InlineSmallCode,          1000);
   product(bool, UseZtso, false, EXPERIMENTAL, "Assume Ztso memory model")        \
   product(bool, UseZihintpause, false, EXPERIMENTAL,                             \
           "Use Zihintpause instructions")                                        \
+  product(bool, UseZvfh, false, "Use Zvfh instructions")                         \
```

Comment:
If adding this flag, please mark as experimental as there is currently no hardware we can test that on (other than QEMU).

Reply:
done